v3/src/components/case-table-card-common/attribute-header.tsx

```diff
@@ -70,6 +70,11 @@ export const AttributeHeader = observer(function ColumnHeader({
 
   useEffect(() => {
     onCloseMenuRef.current?.()
+    if (dragging) {
+      document.body.style.cursor = "grabbing"
+    } else {
+      document.body.style.cursor = "default"
+    }
```

Comment:
I would rather handle this with CSS as we've done elsewhere. For instance, if you assign the `grabbing` cursor style in the `drag-overlay.scss` it should kick in for the drag image.

Reply:
Yes, that works -- thanks 👍